core/storage/leveldb/prefix.cpp

```diff
@@ -151,4 +151,8 @@ namespace fc::storage {
   void OneKey::set(Buffer value) {
     map->put(key, std::move(value)).value();
   }
+
+  void OneKey::remove() {
+    map->remove(key).value();
```

Comment:
why do you get value() here?

Reply:
Why can it fail? How offen will it fail? What can calling code do when it fails?